test/Microsoft.AspNetCore.Sockets.Tests/WebSocketsTests.cs

```diff
@@ -40,7 +40,7 @@ public WebSocketsTests(ITestOutputHelper output)
                     var ws = new WebSocketsTransport(new WebSocketOptions(), connection.Application, connectionContext, loggerFactory);
 
                     // Give the server socket to the transport and run it
-                    var transport = ws.ProcessSocketAsync(await feature.AcceptAsync());
+                    var transport = ws.ProcessSocketAsync(await feature.AcceptAsync(), CancellationToken.None);
```

Comment:
We've generally been using the new [`default` literals](https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-7-1#default-literal-expressions) for this, assuming it works in this position. I do like making it explicit though, so I'm for the change in general :)

Reply:
If it is a true public API then we should default it, but if it is internal then not. Since this will become internal then I'll leave it without default.

Reply:
Nah, I just meant using `default` in the caller. It's just a small style thing to be consistent with how we use do it elsewhere.

Reply:
Meh, I don't really care either way though ;). This is the place where that may be a little odd since it's a little like passing in `null`. We normally used named arguments when passing in uninformative literals like `null`, so this would be `cancellationToken: default`, at which point it might as well be `CancellationToken.None` :)